ios/FluentUI/Command Bar/CommandBarDelegate.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFCommandBarDelegate)
+public protocol CommandBarDelegate: AnyObject {
+    @objc optional func commandBar(_ commandBar: CommandBar, shouldSelectItem item: CommandBarItem) -> Bool
+    @objc optional func commandBar(_ commandBar: CommandBar, didSelectItem item: CommandBarItem)
+    @objc optional func commandBar(_ commandBar: CommandBar, shouldDeselectItem item: CommandBarItem) -> Bool
```

Comment:
How do you expect client to use `shouldSelectItem`? There is already an attribute `isSelected`.  why isn't this as composition as opposed to a `delegate`? 

Also if you need to accept value from client it should be used as a `datasource` as opposed to `delegate`. `Delegate` are intended for callback purposes only.

Reply:
Maybe `UITableViewDelegate` is a good example to follow. In that protocol there are several methods like `func tableView(UITableView, shouldBeginMultipleSelectionInteractionAt: IndexPath) -> Bool` that is used to accept values from client.